skywalking/config.py

```diff
@@ -17,77 +17,73 @@
-                                     os.getenv('SW_AGENT_LOG_REPORTER_IGNORE_FILTER') == 'True' else False  # type: bool
-log_reporter_formatted = False if os.getenv('SW_AGENT_LOG_REPORTER_FORMATTED') and \
-                                  os.getenv('SW_AGENT_LOG_REPORTER_FORMATTED') == 'False' else True  # type: bool
-log_reporter_layout = os.getenv('SW_AGENT_LOG_REPORTER_LAYOUT') or \
-                      '%(asctime)s [%(threadName)s] %(levelname)s %(name)s - %(message)s'  # type: str
-cause_exception_depth = int(os.getenv('SW_AGENT_CAUSE_EXCEPTION_DEPTH') or '5')  # type: int
-
+# Core level configurations
+service_name: str = os.getenv('SW_AGENT_NAME') or 'Python Service Name'
+service_instance: str = os.getenv('SW_AGENT_INSTANCE') or str(uuid.uuid1()).replace('-', '')
+agent_namespace: str = os.getenv('SW_AGENT_NAMESPACE')
+collector_address: str = os.getenv('SW_AGENT_COLLECTOR_BACKEND_SERVICES') or '127.0.0.1:11800'
+kafka_bootstrap_servers: str = os.getenv('SW_KAFKA_REPORTER_BOOTSTRAP_SERVERS') or 'localhost:9092'
+kafka_topic_management: str = os.getenv('SW_KAFKA_REPORTER_TOPIC_MANAGEMENT') or 'skywalking-managements'
+kafka_topic_segment: str = os.getenv('SW_KAFKA_REPORTER_TOPIC_SEGMENT') or 'skywalking-segments'
+kafka_topic_log: str = os.getenv('SW_KAFKA_REPORTER_TOPIC_LOG') or 'skywalking-logs'
+force_tls: bool = os.getenv('SW_AGENT_FORCE_TLS', '').lower() == 'true'
+protocol: str = (os.getenv('SW_AGENT_PROTOCOL') or 'grpc').lower()
+authentication: str = os.getenv('SW_AGENT_AUTHENTICATION')
+logging_level: str = os.getenv('SW_AGENT_LOGGING_LEVEL') or 'INFO'
+disable_plugins: List[str] = (os.getenv('SW_AGENT_DISABLE_PLUGINS') or '').split(',')
+max_buffer_size: int = int(os.getenv('SW_AGENT_MAX_BUFFER_SIZE', '10000'))
+trace_ignore_path: str = os.getenv('SW_TRACE_IGNORE_PATH') or ''
+ignore_suffix: str = os.getenv('SW_IGNORE_SUFFIX') or '.jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,' \
+                                                      '.mp4,.html,.svg '
+correlation_element_max_number: int = int(os.getenv('SW_CORRELATION_ELEMENT_MAX_NUMBER') or '3')
+correlation_value_max_length: int = int(os.getenv('SW_CORRELATION_VALUE_MAX_LENGTH') or '128')
+
+# Plugin configurations
+sql_parameters_length: int = int(os.getenv('SW_SQL_PARAMETERS_LENGTH') or '0')
+pymongo_trace_parameters: bool = True if os.getenv('SW_PYMONGO_TRACE_PARAMETERS') == 'True' else False
+pymongo_parameters_max_length: int = int(os.getenv('SW_PYMONGO_PARAMETERS_MAX_LENGTH') or '512')
+elasticsearch_trace_dsl: bool = True if os.getenv('SW_ELASTICSEARCH_TRACE_DSL') == 'True' else False
+
+http_params_length_threshold: int = int(os.getenv('SW_HTTP_PARAMS_LENGTH_THRESHOLD') or '1024')
+http_ignore_method: str = os.getenv('SW_HTTP_IGNORE_METHOD', '').upper()
+flask_collect_http_params: bool = True if os.getenv('SW_FLASK_COLLECT_HTTP_PARAMS') == 'True' else False
+sanic_collect_http_params: bool = True if os.getenv('SW_SANIC_COLLECT_HTTP_PARAMS') == 'True' else False
```

Comment:
This kind of `bool` expression can be simplified further?

```suggestion
sanic_collect_http_params: bool = os.getenv('SW_SANIC_COLLECT_HTTP_PARAMS') == 'True'
```

Reply:
You r right, 1sec to fix them.